### PR TITLE
Additional item price Model

### PR DESCRIPTION
**Pull request by igorsergiichuk**
## Overview

| Q                 | A
| ----------------- | ---
| Bugfix?           | no
| New feature?      | yes
| Breaking Changes? | no
| Fixed issues      | #

## The problem

AdditionalItemPrice Model was absent

## How I resolved it

Added AdditionalItemPrice

## How to verify it

Test it

## Notes

Check the delete tests please

Notify the following people: @darrynten

### CODE REVIEW

File: src/Models/AdditionalItemPrice.php [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    ];
+
+    /**
+     * @var array $features
+     */
+    protected $features = [
+        'all' => true,
+        'get' => true,
+        'save' => true,
+        'delete' => true
```
Comment (darrynten) [inline src/Models/AdditionalItemPrice.php]: Missing a trailing comma here :)

File: tests/SageOne/Models/AdditionalItemPriceModelTest.php [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+            $this->assertEquals($model->itemId, 2);
+            $this->assertEquals($model->priceInclusive, 3.0);
+            $this->assertEquals($model->priceExclusive, 4.0);
+            $this->assertEquals($model->additionalPriceListId, 5.0);
+        });
+    }
+
+    public function testDelete()
+    {
+        $this->verifyDelete(AdditionalItemPrice::class, 1, function ($model) {
```
Comment (darrynten) [inline tests/SageOne/Models/AdditionalItemPriceModelTest.php]: See new way of testing delete in #68

**Review by darrynten — CHANGES_REQUESTED** [review]

File: tests/mocks/AdditionalItemPrice/GET_AdditionalItemPrice_Get.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+      "PriceInclusive": 3.0,
+      "PriceExclusive": 4.0,
+      "AdditionalPriceListId": 5
+    },
+    {
+      "ID": 2,
+      "ItemId": 2,
+      "PriceInclusive": 3.0,
+      "PriceExclusive": 4.0,
+      "AdditionalPriceListId": 5
```
Comment (fergusdixon) [inline tests/mocks/AdditionalItemPrice/GET_AdditionalItemPrice_Get.json]: All fields need to be unique, update these and the mocks from #72 in this pr please

**Review by fergusdixon — CHANGES_REQUESTED** [review]
Just some changes to mocks for this model and the model from #72

File: README.md [hunk 4]
```diff
@@ -293,6 +293,7 @@ Models marked with an asterix are pure CRUD models
     - [x] Account Opening Balance *
     - [x] Account Payment *
     - [x] Account Receipt *
+  - [x] Additional Item Price *
```
Comment (darrynten) [inline README.md:296]: It's great that you added it here, but you must also remember to remove it from here:

https://github.com/darrynten/sage-one-php/pull/73/files#diff-04c6e90faac2675aa89e2176d2eec7d8R342

**Review by darrynten — CHANGES_REQUESTED** [review]
Minor change request

File: README.md (hunk 4 above)
Comment (igorsergiichuk) [reply README.md:296]: Done

**Review by darrynten — APPROVED** [review]

**Review by fergusdixon — APPROVED** [review]
Good work!

**Pull request merged**